tests/Middleware/SanitizeHttpHeadersMiddlewareTest.php

```diff
@@ -25,10 +26,17 @@ public function testInvoke($inputData, $expectedData)
     {
         $event = new Event(new Configuration());
         $event->setRequest($inputData);
+        $request = $this->prophesize(ServerRequestInterface::class)->reveal();
 
         $callbackInvoked = false;
-        $callback = function (Event $eventArg) use ($expectedData, &$callbackInvoked) {
+        $callback = function (
+            Event $eventArg,
+            ServerRequestInterface $passedRequest = null,
+            $exception = null,
```

Comment:
Even though this is the correct function definition for a middleware, I don't see any usage of the `$exception` and `$payload` arguments so I think we can safely remove them and put everything on one line for better readability. Otherwise (I favour this) I would use use the moment to change all tests to verify that the middlewares pass all the correct arguments to the next one

Reply:
Good suggestion. I'll refactor the callback elsewhere then.